pkg/executor/executortype/newdeploy/newdeploymgr.go

```diff
@@ -260,7 +260,7 @@ func (deploy *NewDeploy) RefreshFuncPods(ctx context.Context, logger *zap.Logger
 		UID:       env.ObjectMeta.UID,
 	})
 
-	dep, err := deploy.kubernetesClient.AppsV1().Deployments(metav1.NamespaceAll).List(ctx, metav1.ListOptions{
+	dep, err := deploy.kubernetesClient.AppsV1().Deployments(f.Spec.Environment.Namespace).List(ctx, metav1.ListOptions{
```

Comment:
Lets keep this NamespaceAll for now.

Reply:
We will have identify namespace for deployement based the environment here. If environment is default namespace deployment would get created in fission-function namespace.

Reply:
done